### PR TITLE
Clients: print to print() in rucio/rse and rucio/rse/protocols #819

### PR DESCRIPTION
**Pull request by jwackito**
Clients: print to print() in rucio/rse and rucio/rse/protocols #819

### CODE REVIEW

File: lib/rucio/rse/protocols/gsiftp.py [hunk 1]
```diff
@@ -14,6 +14,8 @@
  - Mario Lassnig, <mario.lassnig@cern.ch>, 2017
```
Comment (tbeerman) [inline lib/rucio/rse/protocols/gsiftp.py:14]: Add author!

File: lib/rucio/rse/protocols/signeds3.py [hunk 2]
```diff
@@ -9,6 +9,8 @@
 # - Wen Guan, <wen.guan@cern.ch>, 2016-2017
 # - Mario Lassnig, <mario.lassnig@cern.ch>, 2017
```
Comment (tbeerman) [inline lib/rucio/rse/protocols/signeds3.py:17]: Add author!

File: lib/rucio/rse/rsemanager.py [hunk 3]
```diff
@@ -16,6 +16,8 @@
 # - Frank Berghaus, <frank.berghaus@cern.ch>, 2018
```
Comment (tbeerman) [inline lib/rucio/rse/rsemanager.py]: Add author!

**Review by tbeerman — DISMISSED** [review]

File: lib/rucio/rse/protocols/gsiftp.py (hunk 1 above)
Comment (vingar) [reply lib/rucio/rse/protocols/gsiftp.py:14]: you can even use the script:  ./tools/add_header {file} which will update the author list and put the right header

**Comment by bari12** [comment]
The failed tests are the usual pylint errors due to gfal
Can you try to disable this import error @jwackito

File: lib/rucio/rse/protocols/gsiftp.py [hunk 4]
```diff
@@ -12,8 +12,11 @@
  - Tomas Javurek, <Tomas.Javurek@cern.ch>, 2016
  - Cedric Serfon, <cedric.serfon@cern.ch>, 2016-2017
  - Mario Lassnig, <mario.lassnig@cern.ch>, 2017
+ - Joaquin Bogado, <jbogado@linti.unlp.edu.ar> 2018
```
Comment (bari12) [inline lib/rucio/rse/protocols/gsiftp.py]: , 2018

**Review by bari12 — CHANGES_REQUESTED** [review]

File: lib/rucio/rse/rsemanager.py [hunk 5]
```diff
@@ -14,7 +14,9 @@
 # - Martin Barisits, <martin.barisits@cern.ch>, 2017-2018
 # - Tobias Wegner, <tobias.wegner@cern.ch>, 2017
 # - Frank Berghaus, <frank.berghaus@cern.ch>, 2018
+# - Joaquin Bogado, <jbogado@linti.unlp.edu.ar> 2018
```
Comment (bari12) [inline lib/rucio/rse/rsemanager.py]: The comma is still missing here JJ 😆

Comment (vingar) [reply lib/rucio/rse/rsemanager.py]: just run:
```
./tools/add_header -i lib/rucio/rse/rsemanager.py 
```

**Review by bari12 — APPROVED** [review]

**Pull request merged**